ui/components/component-library/button-link/button-link.scss

```diff
@@ -1,5 +1,10 @@
-
 .mm-button-link {
+  &:hover:not(&--disabled) {
+    text-decoration: underline;
+    text-decoration-thickness: 2px;
+    text-underline-offset: 4px;
```

Comment:
Forgive me if I missed this discussion but what's the reason to have different thicknesses across sizes? I was expecting it just to be the default across the board. I kind of like the distinction between button and link though

Reply:
Ah right can you add the figma link or screenshot to the description I can't find it
![Screenshot 2023-07-13 at 4 19 42 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/7bd46aa4-1560-424b-8441-4feaf4f7f9d9)